src/docs/asciidoc/en/user-guide/10_using_the_disruptor.adoc

```diff
@@ -449,7 +449,7 @@ However, again knowledge of the deployed system can allow for additional perform
 
 - **SleepingWaitStrategy** ->
 --
-Like the `BlockingWaitStrategy` the `SleepingWaitStrategy` it attempts to be conservative with CPU usage, by using a simple busy wait loop, but uses a call to link:http://docs.oracle.com/javase/7/docs/api/java/util/concurrent/locks/LockSupport.html#parkNanos(long)[`LockSupport.parkNanos(1)`^] in the middle of the loop.
+Like the `BlockingWaitStrategy` the `SleepingWaitStrategy` it attempts to be conservative with CPU usage, by using a simple busy wait loop, but uses a call to link:https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/locks/LockSupport.html#parkNanos(long)[`LockSupport.parkNanos(1)`^] in the middle of the loop.
```

Comment:
Changed from JDK 1.5 -> 8 docs, but we're Disruptor 4.0 will be changing JDK version from 7 -> 11, so maybe pick the JDK 11 docs?